packages/nextjs/components/MetaHeader.tsx

```diff
@@ -16,7 +16,7 @@ const baseUrl = process.env.NEXT_PUBLIC_VERCEL_URL ? `https://${process.env.NEXT
 export const MetaHeader = ({
   title = "Scaffold-ETH 2 - Open source toolkit to build dApps on Ethereum",
   description = "An open-source, up-to-date toolkit for building decentralized applications (dapps) on the Ethereum blockchain. It's designed to make it easier for developers to create and deploy smart contracts and build user interfaces that interact with those contracts.",
-  image = "thumbnail.jpg",
+  image = "og-image.png",
```

Comment:
(looking at this with Shiv and Pablo, live on ETH London :D)

We like your thumbnail more! Could you create the same for the Twitter image with the `twitterThumbnail.jpg` dimensions?